### ISSUE TITLE
Add a section explaining how to use nearby feature in the tutorial

### ISSUE THREAD

**Issue by Anu-123-gif** [commons-app/apps-android-commons#4262]
**Summary:** 

It is quite confusing to use the nearby feature and it would be very helpful for new users if the tutorial explained this as well.

**Would you like to work on the issue?**
Yes
With some guidance :)
Please let us know whether you want to fix the issue by yourself. If not, anyone can get the issue assigned to them.

**Comment by nicolas-raoul** [comment]
I agree that we would need documentation about Nearby.

However, I don't think the tutorial is the best place. The tutorial is for ultra-important policy explanation (copyright, and what kind of pictures are expected). The tutorial is not meant to explain anything technical (features, buttons, UI) as it would make it too long. The tutorial is to prevent behaviors that would get us into big trouble.

How about a walk-through at https://github.com/commons-app/commons-app-documentation/blob/master/android/How-to-install-and-use-the-app.md ?

**Comment by Anu-123-gif** [comment]
Yes, making the tutorial too long won't really help.
A documentation sounds good!
We could add a link to the documentation in the tutorial if the users want to know more about Nearby.

**Comment by nicolas-raoul** [comment]
Of course, please feel free to add a new section at https://github.com/commons-app/commons-app-documentation/blob/master/android/How-to-install-and-use-the-app.md I believe you have edit rights, please let us know if not.

I agree that a "User guide" link in the hamburger menu could be a good idea. Please note that it would be an "enhancement" rather than a "bug", though, which has an impact on who can implement it per https://github.com/commons-app/commons-app-documentation/blob/master/android/Volunteers-welcome!.md#developers

Cheers!

**Comment by nicolas-raoul** [comment]
Ah, sorry, my link was wrong, the documentation is now here: https://commons-app.github.io/docs.html
